packages/react-ui/components/Kebab/__tests__/Kebab.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { Kebab } from '../Kebab';
+
+describe('Kebab', () => {
+  it('calls onFocus callback', () => {
+    const onFocus = jest.fn();
+    const kebab = mount(<Kebab onFocus={onFocus} />);
+
+    kebab.find('[tabIndex=0]')
```

Comment:
Здесь можно использовать иконку в качестве селектора:
```suggestion
    kebab.find(MenuKebabIcon)
```

Это более однозначный и "устойчивый" поиск.